### DISCUSSION TITLE
2020/06/29/oneplus-8-pro-volte-vowifi/

### DISCUSSION THREAD

**Discussion by giscus** [jdeluyck/kcoreorg-website#142] [Blog Comments]
# 2020/06/29/oneplus-8-pro-volte-vowifi/

Orange, my ISP, supports VoLTE (Voice over LTE) and  VoWiFi (Voice over WiFi) on a  limited set of devices. Unfortunately, OnePlus is not one of the manufacturers.

https://kcore.org/2020/06/29/oneplus-8-pro-volte-vowifi/

**Comment by ranareehanaslam** [comment]
Follow up the easy guide how to enable VOLT and wifi calling on any OnePlus  Device

https://youtu.be/H9eXj2CsCiw

**Comment by jdeluyck** [comment]
FYI: this does not work if your provider does not provision VoLTE/VoWifi.
